.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ executors:
   node-executor:
     working_directory: ~/widgets
     docker:
-      - image: circleci/node:fermium-browsers
+      - image: cimg/node:20.13.1-browsers
```

Comment:
![Screenshot 2024-07-18 at 1 04 42 PM](https://github.com/user-attachments/assets/cf67cf5f-f723-401c-bb7b-0ac167e48ec1)
fermium was node-14. I think iron would be 20. Do you think we use iron here instead of the version.

Reply:
I tried using iron, but it didn't work. I couldn't find circle ci image name with node:iron-browsers